docs/user/dql/functions.rst

```diff
@@ -1557,6 +1585,31 @@ Example::
     +------------------------------+
 
 
+UNIX_TIMESTAMP
+--------------
+
+Description
+>>>>>>>>>>>
+
+Usage: Converts given argument to Unix time (seconds since Epoch - very beginning of year 1970). If no argument given, it current Unix time.
```

Comment:
```suggestion
Usage: Converts given argument to Unix time (seconds since January 1st, 1970 at 00:00:00 UTC). If no argument given, it returns current Unix time.
```

Reply:
Thank you, updated in dfd2c1ea.